recipes-ni/ni-rtfeatures/ni-rtfeatures_1.0.bb

```diff
@@ -0,0 +1,62 @@
+SUMMARY = "rtfeatures user-space tools"
+DESCRIPTION = "Provides user-space tools to support rtfeatures."
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+LICENSE = "MIT"
+SECTION = "base"
+
+DEPENDS += "\
+	update-rc.d-native \
+"
+
+SRC_URI += "\
+	file://init.d/handle_cpld_ip_reset \
+	file://rtfeatures.rules \
+"
+
+S = "${WORKDIR}"
+
+
+do_install () {
+	install -d ${D}${sysconfdir}/init.d/
+
+	install -m 0755 ${S}/init.d/handle_cpld_ip_reset    ${D}${sysconfdir}/init.d
+
+	install -d ${D}${sysconfdir}/udev/rules.d
+
+	install -m 0644 ${S}/rtfeatures.rules    ${D}${sysconfdir}/udev/rules.d/rtfeatures.rules
+}
+
+pkg_postinst_${PN} () {
+	if [ -n "$D" ]; then
+		OPT="-r $D"
+	else
+		OPT="-s"
+	fi
+
+	update-rc.d $OPT handle_cpld_ip_reset    start 6 1 3 4 5 .
+}
+
+pkg_postrm_${PN} () {
+	if [ -n "$D" ]; then
+		OPT="-f -r $D"
+	else
+		OPT="-f"
+	fi
+
+	update-rc.d $OPT handle_cpld_ip_reset  remove
+}
```

Comment:
The `ni-hw-support-scritps*` recipes explicitly defined these functions because the OE-core `update-rc.d` bbclass doesn't support installing multiple initscripts within the same package (which the generic hw-support recipes needed to do).

Now that we have broken this feature out into its own recipe, I don't see us needed (or wanting) to install multiple initscripts, so lets prefer to use that bbclass instead.

Reply:
That makes sense, inherit from `update-rc.d` instead.